_posts/2022-02-23-roadmap-proposal.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: "Update: OpenSearch Proposed 2022 Release Schedule"
+authors:
+  - henkle
+date: 2022-02-23
+categories:
+  - partners
+twittercard:
+  description: "As part of thinking about releases for this year, I’ve been trying to sketch out a schedule of major and minor release dates through 2023. I wanted to share with you what I have so far, and hear your thoughts."
+excerpt: "As part of thinking about releases for this year, I’ve been trying to sketch out a schedule of major and minor release dates through 2023. I wanted to share with you what I have so far, and hear your thoughts."
+
+---
+
+As part of thinking about releases for this year, I’ve been trying to sketch out a schedule of major and minor release dates through 2023. I wanted to share with you what I have so far. 
+
+### Considerations
+
+Before getting to the dates themselves, here are the things I was thinking about when I picked them. 
+
+1. **The OpenSearch Project uses SemVer.** 
+As a reminder, the OpenSearch project assigns version numbers to releases following [the semantic versioning convention](https://opensearch.org/blog/technical-post/2021/08/what-is-semver/).  This means the project won’t introduce an incompatible change without also incrementing the major version number.  
+
+2.  **Everyone is excited about Lucene 9.0.0.**
+Lucene 9.0.0 [released](https://lucene.apache.org/core/corenews.html#apache-lucenetm-900-available) on December 7, 2021. It includes several new features and performance improvements that the team would like to make available to users of OpenSearch, including K-NN support, Vectors, Big Endian, Faster numeric indexing, faster sorting, concurrent merge scheduler, and prototype Java Jigsaw module support. It will take a few releases for OpenSearch to take full advantage of it, but the potential of Lucene 9.0.0 is so exciting getting support in as soon as possible is the priority over trying to bundle it with other changes.
+
+3.  **The project will follow a "release train" model for minor-version releases, and a "tentpole" model for major-version releases.**
+The team wants to produce minor-version releases on a "release-train" schedule: every six weeks a new release comes out, and it includes all the new features that are ready at the time of release.  Having a set release schedule makes sure OpenSearch is releasing in a predictable way and prevents a backlog of unreleased changes. In contrast, major-version releases will take place when there is a critical mass of new features that would create incompatibilities, since that’s the only opportunity to release them. These are called "tentpole" features because getting one of these in might hold a release. If any of the tentpole changes runs into serious problems, a decision would need to be made to move the release or wait until the next major release for that change. 
+
```

Comment:
That passive voice in the last sentence raises more questions than it answers, I think?

If any of the tentpole changes runs into serious problems, the folks working on it will need to add a comment on their meta tracking issue with an alert.  At that point we the folks working on the issue and I can decide whether to cut scope, remove the feature from the release, or move the release date depending on the type and severity of the problem.

Reply:
I'll sub in.

Reply:
"folks" is weird I acknowledge, BTW.  So if you have a better word feel free to swap it in (tent pole features can come from anywhere.

Reply:
Contributors?